fairscale/experimental/tests/nn/ampnet_pipe_process/test_ampnet_pipe.py

```diff
@@ -17,6 +17,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import pytest
```

Comment:
I am confused, is this file in fairscale dir or in tests dir? github shows that it is in fairscale dir, but should it be in tests/experimental/...

Reply:
Thanks for catching this! It was a cached file in my editor that I made changes to by mistake.